projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/select-docs/select-docs.component.html

```diff
@@ -298,6 +298,33 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
         </div>
       </div>
 
+      <div class="go-column go-column--100 go-column--no-padding">
+        <div class="go-container">
+          <div class="go-column go-column--100">
+            If need be, you can also remove the "Select All" button:
```

Comment:
Think it might be worth adding "with the `@Input() showSelectAll: boolean = true;` binding"? That would be more consistent with the others and might make it a bit easier to spot (I had a little trouble finding it as I was scanning down the page).

Reply:
Sure, I added that in.